packages/scene-composer/src/utils/entityModelUtils/updateNodeEntity.ts

```diff
@@ -44,6 +46,20 @@ export const updateEntity = async (
     },
   };
 
+  if (sceneRootEntityId) {
+    const oldParentEntityId = oldParentRef ?? sceneRootEntityId;
+    const newParentEntityId = node.parentRef ?? sceneRootEntityId;
+    if (newParentEntityId !== newParentEntityId) {
```

Comment:
Looks like a cut and paste error here.  Should be oldParentEntityId !== newParentEntityId, not new !== new.

Reply:
Good catch. File updated.